WordPressCom-Stats-iOS/UI/NSObject+StatsBundleHelper.m

```diff
@@ -0,0 +1,13 @@
+#import "NSObject+StatsBundleHelper.h"
+
+@implementation NSObject (StatsBundleHelper)
+
+- (NSBundle *)statsBundle
+{
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"WordPressCom-Stats-iOS" ofType:@"bundle"];
```

Comment:
And since we're at it, I'd make StatsBundleHelper a class. It feels kind of weird to extend NSObject for this, since it also extends all non-stats classes.